### PR TITLE
Add zazu-dev-doc plugin to zazu plugin directory

### PR DESCRIPTION
**Pull request by jgoday**

### CODE REVIEW

File: docs/_packages/zazu-dev-doc.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+top:         false
+type:        plugin
+title:       "Zazu dev documentation"
```
Comment (bayleedev) [inline docs/_packages/zazu-dev-doc.md]: Can this be "Dev Documentation" instead?

Comment (jgoday) [reply docs/_packages/zazu-dev-doc.md]: Ok, I already changed it !

**Comment by jgoday** [comment]
Changed plugin doc file title

File: docs/_packages/zazu-dev-doc.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+top:         false
+type:        plugin
+title:       "Zazu Dev Documentation"
```
Comment (bayleedev) [inline docs/_packages/zazu-dev-doc.md]: Sorry I should clarify... I meant without "Zazu" in the name.

**Comment by jgoday** [comment]
Changed plugin doc file title

**Pull request merged**